go/vt/vtexplain/testdata/multi-output/selectsharded-output.txt

```diff
@@ -137,10 +137,10 @@ select name from user where exists (select id from t1) /* non-correlated subquer
 ----------------------------------------------------------------------
 select * from name_info order by info /* select * and order by varchar column */
 
-1 ks_sharded/-40: select `name`, info, weight_string(info) from name_info order by info asc limit 10001 /* select * and order by varchar column */
-1 ks_sharded/40-80: select `name`, info, weight_string(info) from name_info order by info asc limit 10001 /* select * and order by varchar column */
-1 ks_sharded/80-c0: select `name`, info, weight_string(info) from name_info order by info asc limit 10001 /* select * and order by varchar column */
-1 ks_sharded/c0-: select `name`, info, weight_string(info) from name_info order by info asc limit 10001 /* select * and order by varchar column */
+1 ks_sharded/-40: select `name`, info from name_info order by info asc limit 10001 /* select * and order by varchar column */
+1 ks_sharded/40-80: select `name`, info from name_info order by info asc limit 10001 /* select * and order by varchar column */
+1 ks_sharded/80-c0: select `name`, info from name_info order by info asc limit 10001 /* select * and order by varchar column */
+1 ks_sharded/c0-: select `name`, info from name_info order by info asc limit 10001 /* select * and order by varchar column */
```

Comment:
No need for `weight_string` anymore here!

Reply:
Because we now pass through the correct type information in more cases. This means we don't need to query for the weight string anymore since we know the collation of the results here. Before we missed that information and vtgate could not sort things correctly and therefore asked for the weight string.